static/elements/chromedash-form-field.js

```diff
@@ -1,8 +1,92 @@
-import {LitElement, html} from 'lit';
+import {LitElement, html, css} from 'lit';
 import {ALL_FIELDS} from './form-field-specs';
 import {unsafeHTML} from 'lit/directives/unsafe-html.js';
+import {SHARED_STYLES} from '../sass/shared-css.js';
+
 
 export class ChromedashFormField extends LitElement {
+  static get styles() {
```

Comment:
I'm not sure this change to chromedash-form-field.js is correct, or maybe it is simply unnecessary.  Previously I changed the form-field html to render in the light dom, so the styles in this component, which are automatically qualified to apply only to this component, will not apply.  Instead, the same styles were copied to the forms.scss file.

Reply:
I see. Thank you for the clarification. I was wondering why the styles are not applied. I will moved these back to form.scss.

Reply:
I moved these (chromedash-form-field related css) back to form.scss, and also put them in form.css.js.